src/algebra/big_operators/basic.lean

```diff
@@ -1855,28 +1855,50 @@ end
 
 /-! ### `additive`, `multiplicative` -/
 
-namespace finset
 open additive multiplicative
 
 section comm_monoid
-variables [comm_monoid β]
+variables [comm_monoid α]
+
+@[simp] lemma of_mul_list_prod (s : list α) : of_mul s.prod = (s.map of_mul).sum :=
+by simpa [of_mul]
+
+@[simp] lemma to_mul_list_sum (s : list (additive α)) :
+  to_mul s.sum = (s.map to_mul).prod := by simpa [to_mul, of_mul]
```

Comment:
List lemmas shouldn't need commutativity.

Reply:
Of course :face_palm: